stat.h

```diff
@@ -237,6 +238,12 @@ struct thread_stat {
 	fio_fp64_t ss_deviation;
 	fio_fp64_t ss_criterion;
 
+	uint32_t priority_bit;
+	uint64_t io_u_plat_prio[FIO_IO_U_PLAT_NR];
+	uint64_t io_u_plat_low_prio[FIO_IO_U_PLAT_NR];
+	struct io_stat clat_prio_stat;
+	struct io_stat clat_low_prio_stat;
+
```

Comment:
The stat changes would need client/server changes as well, see how we convert values across the wire in client.c and server.c